code/controllers/configuration.dm

```diff
@@ -529,6 +535,58 @@
 	if(fps <= 0)
 		fps = initial(fps)
 
+
+/datum/configuration/proc/loadmaplist(filename)
+	var/list/Lines = file2list(filename)

```

Comment:
one, too much work
two, too much work
three, with the amount of conditions i wanted to have, regex wouldn't have worked.